gvsbuild/tools.py

```diff
@@ -28,7 +28,8 @@ def __init__(self):
         Tool.__init__(
             self,
             "cargo",
-            version="1.81.0",
+            # Linking error with the latest Cargo version and librsvg
```

Comment:
Is there an upstream ticket?

Reply:
It is the same issue as this one, although the ticket is closed: https://gitlab.gnome.org/GNOME/librsvg/-/issues/1107

Reply:
mmm, I am not super happy about being stuck with this forever :(

Can you split the librsvg update in its own PR so that we land the rest of the updates in the meantime?
While you are at it, link that issue in the comment so that we can more easily refer back to it

Reply:
Hi @pbor, I split the librsvg changes out and created another PR.